src/components/TodoListItem.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react'
 
-const TodoListItem = ({todo,onRemoveTodo}) => {
-  console.log({todo})
-  return(
+const TodoListItem = ({ todo, onRemoveTodo }) => {
+  console.log({ todo })
+  return (
     <div>
-      <li> {todo.title} <button type="button" onClick = {() => onRemoveTodo(todo.id)}><span>&nbsp;</span>Remove </button>
-      </li>       
+      <li> {todo.fields.Title} <button type="button" onClick={() => onRemoveTodo(todo.id)}><span>&nbsp;</span>Remove </button>
```

Comment:
There are a couple of unnecessary spaces here, and also properties like `.Title` should generally be lowercase. Good spacing may seem unimportant, but it can make things more difficult to understand, and sometimes even introduce  bugs that are extremely difficult to track down, so it's good to get into a habit of paying attention to good spacing early on.